packages/@react-aria/listbox/src/useListBox.ts

```diff
@@ -34,11 +38,10 @@ interface AriaListBoxProps {
    */
   keyboardDelegate?: KeyboardDelegate,
 
-  /** Whether the auto focus the listbox or an option. */
-  autoFocus?: boolean | FocusStrategy,
-
-  /** Whether focus should wrap around when the end/start is reached. */
-  shouldFocusWrap?: boolean
+  /**
+   * An optional visual label for the listbox.
+   */
+  label?: ReactNode
```

Comment:
This prop is only supported in react-aria. Spectrum doesn't have a visual label for listbox